main.js

```diff
@@ -0,0 +1,9 @@
+for (let i = 1; i <= 99; ++i) {
+  if (i % 15 === 0) {
+    console.log("FizzBuzz" + i);
```

Comment:
一点修正のご連絡と、ご質問の回答もさせていただきます！

【修正点】
JSでは文字列を作成する際、、`""` ではなく `''`を使用する慣習があります。
```suggestion
    console.log('FizzBuzz' + i);
```
どちらでも動作はするのですが、JSの慣習ということで、今後は`''`を使用するクセをつけると良いと思います！

=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
【ご質問の回答】
スペースを文字列として入れる、という方法で大丈夫です！
```suggestion
    console.log("FizzBuzz " + i);
```

別解として、[テンプレートリテラル](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals)を使用すると一文で記述できたりします。
```suggestion
    console.log(`FizzBuzz ${i}`);
```

テンプレートリテラルは、変数を使用した文字列を作成する際によく使用される記述方法となりますので、覚えておくと役に立ちます！